src/components/Mailbox.vue

```diff
@@ -129,7 +129,7 @@ export default {
 		},
 		envelopesToShow() {
 			if (this.paginate === 'manual' && !this.expanded) {
-				return this.envelopes.slice(0, 8)
+				return this.envelopes.slice(0, 6)
```

Comment:
A bit smelly that you have to adjust the same magic number in two paces, isn't it? Replace this occurrence by `this.initialPageSize`

Reply:
> Replace this occurrence by this.initialPageSize

thats true, will do. But i still dont see how i can change the number on envelopes shown on important section depending on the screen, when we have a initialization of 6 envelopes and a 'manual' pagination/loadmore

Reply:
Derive a guesstimate from `window.innerHeight`